<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Mulish</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Interpolator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="csvLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectInputButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenTableButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string>Generate Table</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImportLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ExportLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveInterpolatedDataButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string>Save Interpolated Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="InterpolatedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveDailyDataButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string>Save Daily Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="DailyDataLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveHHTableButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mulish</family>
     </font>
    </property>
    <property name="text">
     <string>Save HH Table</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
